app/env.js

```diff
@@ -30,6 +30,8 @@ const vars = envalid.cleanEnv(
     LOG_LEVEL: envalid.str({ default: 'info', devDefault: 'debug' }),
     PORT: envalid.port({ default: 80, devDefault: 3000 }),
     NODE_HOST: envalid.host({ devDefault: 'localhost' }),
+    IPFS_API_HOST: envalid.host({ devDefault: 'localhost' }),
+    IPFS_API_PORT: envalid.host({ devDefault: '5001' }),
```

Comment:
yea, it wasnt used before, good point, but again this is not AC. will update but i think issues like this could be also abstracted and converted into 'ascetics stories'

Reply:
updated, also spotted another one:) `    IPFS_API_PORT: envalid.host({ default: '5001' }),`